Docs improvements for develop.rst (#18191)
scikit-learn/scikit-learn: doc/developers/develop.rst

    from sklearn.utils import check_array, check_random_state

    def choose_random_sample(X, random_state=0):
        """Choose a random point from X.

        Parameters
        ----------
        X : array-like of shape (n_samples, n_features)
            An array representing the data.

>         random_state : RandomState or int, default=0

Reviewer: ```suggestion
        random_state : int or RandomState instance, default=0
```

just for consistency with other docstrings in the documentation

    def choose_random_sample(X, random_state=0):

>         """
>         Choose a random point from X.

Reviewer 2: ```suggestion
        """Choose a random point from X.
```

Author: Thanks, I haven't noticed it.

        Parameters
        ----------
        X : array-like of shape (n_samples, n_features)
            An array representing the data.
        random_state : RandomState or int, default=0

>             A random number generator instance or an int seed to define
>             the state of the random permutations generator.

Reviewer 2: ```suggestion
            The seed of the pseudo random number generator that selects a
            random sample. Pass an int for reproducible output across multiple
            function calls.
            See :term:`Glossary <random_state>`.
```

Author: Ok
